tensorflow/python/ops/ctc_ops.py

```diff
@@ -47,6 +47,18 @@ def ctc_loss(inputs, labels, sequence_length,
   max(labels.indices(labels.indices[:, 1] == b, 2))
     <= sequence_length(b) for all b.
   ```
+  
+  Useful Tips:
+  
+  The ctc_loss has as input an affine projection of RNN state:
+    inputs_t = W*h^L_{t} + b, for t = {1,...,max_time}
+  where h is the hidden state of last rnn calculated at timestep t.
+  
+  The `inputs` argument has size num_labels + 1 where +1 is reserved to
```

Comment:
**Note**: The `inputs` Tensor's innermost dimension size, `num_classes`, represents
`num_labels + 1` classes, where `num_labels` is the number of true labels, and the
largest value (`num_classes - 1`) is reserved for the blank label.

For example, for a vocabulary containing 3 labels `[a, b, c]`, `num_classes = 4` and the
labels indexing is `{a: 0, b: 1, c: 2, blank: 3}`.